src/gmt_plot.c

```diff
@@ -2032,43 +2032,43 @@ GMT_LOCAL void plot_map_tickitem (struct GMT_CTRL *GMT, struct PSL_CTRL *PSL, do
 				ny = gmtlib_linear_array (GMT, s, n, dy, GMT->current.map.frame.axis[GMT_Y].phase, &val);
 		}
 		for (i = 0; i < ny; i++) {
-			if (skip_polar_apex_annotation (GMT, i, val, ny)) continue;
-			shift = plot_shift_gridline (GMT, val[i], GMT_Y);
-			plot_map_lattick (GMT, PSL, val[i] + shift, w, e, len);
+			if (gmtplot_skip_polar_apex_annotation (GMT, i, val, ny)) continue;
+			shift = gmtplot_shift_gridline (GMT, val[i], GMT_Y);
+			gmtplot_map_lattick (GMT, PSL, val[i] + shift, w, e, len);
 		}
 		if (ny) gmt_M_free (GMT, val);
 	}
 
 	GMT->current.map.on_border_is_outside = false;	/* Reset back to default */
 }
 
-GMT_LOCAL void plot_map_tickmarks (struct GMT_CTRL *GMT, struct PSL_CTRL *PSL, double w, double e, double s, double n) {
+GMT_LOCAL void gmtgmtplot_map_tickmarks (struct GMT_CTRL *GMT, struct PSL_CTRL *PSL, double w, double e, double s, double n) {
```

Comment:
There area a couple of gmtgmt....

Reply:
Yes, I got those too for the same reasons.  Good to grep for gmtgmt as a check.

Reply:
Fixed.